### ISSUE TITLE
Change position of the resume button to a more accessible position.

### ISSUE THREAD

**Issue by THElegend5** [tachiyomiorg/tachiyomi#5974]
### Describe your suggested feature

Right now the position of resume button is not the best , you have to go all the way down to reach it. Which just makes it so that you don't want to use it.
A better position for it could be 
![IMG_20210927_094816](https://user-images.githubusercontent.com/63903377/134844847-bafa6427-cd19-4e86-8a5d-f622e2e282ec.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.12.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by THElegend5** [comment]
Also if after changing the position to the one I suggested if after scrolling it goes up to another not so great position. Maybe it can be made so that it goes to the current position after scrolling a certain amount.

**Comment by Vacui** [comment]
I think that putting the resume button in the bottom right corner is a decision made by the designer to support the vertical layout of the app.

In-app design there is a test called **Thumb Zone**, and it is based on where people can more easily reach on the screen using just one finger. As you see in the image below, the bottom right corner is the best spot. In modern app design more often you also see a preference for the center spot, to satisfy both right and left-handed.
The position you suggested would be in a good spot, but as you already spotted, many problems would come with that change, making it not convenient.
![image](https://user-images.githubusercontent.com/45497567/134889840-ff34ac6c-5d4a-4c26-ae1c-2f3b88c69dac.png)

This is just my opinion, and I hope it helped.

Have a great day,
Matteo

**Comment by THElegend5** [comment]
> I think that putting the resume button in the bottom right corner is a decision made by the designer to support the vertical layout of the app.
> 
> In-app design there is a test called **Thumb Zone**, and it is based on where people can more easily reach on the screen using just one finger. As you see in the image below, the bottom right corner is the best spot. In modern app design more often you also see a preference for the center spot, to satisfy both right and left-handed.
> The position you suggested would be in a good spot, but as you already spotted, many problems would come with that change, making it not convenient.
> ![image](https://user-images.githubusercontent.com/45497567/134889840-ff34ac6c-5d4a-4c26-ae1c-2f3b88c69dac.png)
> 
> This is just my opinion, and I hope it helped.
> 
> Have a great day,
> Matteo

Yeah I am right handed and as you can see in the diagram you shared the current position is not ideal. I think it would be better  even if instead of what I earlier proposed if the position was moved from bottom right to bottom center.

**Comment by THElegend5** [comment]
Specially in the bigger phones where you don't want the position of such functions which would help a lot if they were easily accessible to be on the extremes.

**Comment by arkon** [comment]
No plans to change this. FABs in the bottom right is a standard pattern for phones.

"Buttons: floating action button - Material Design" https://material.io/components/buttons-floating-action-button
